src/index.html

```diff
@@ -4,9 +4,548 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css"
+      integrity="sha512-NmLkDIU1C/C88wi324HBc+S2kLhi08PN5GDeUVVVC/BVt/9Izdsc9SVeVfA1UZbY3sHUlDSyRXhCzHfr6hmPPw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../src/styles/index.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__conteiner">
+        <div class="header__logo">
+          <a href="#" class="header__linkLogo">
+            <img
+              src="./images/logo.png"
+              alt="Logo"
+              class="header__logoImg"
+            >
+          </a>
+        </div>
+        <nav class="navigation">
+          <ul class="navigation__list">
+            <li class="navigation__item">
+              <a href="#" class="navigation__link is-active">APPLE</a>
+            </li>
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">samsung</a>
+            </li>
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">SMARTPHONES</a>
+            </li>
+            <li class="navigation__item">
+              <a
+                href="#"
+                class="navigation__link"
+                data-qa="nav-hover"
+              >
+                LAPTOPS & COMPUTERS
+              </a>
+            </li>
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">GADGETS</a>
+            </li>
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">TABLETS</a>
+            </li>
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">PHOTO</a>
+            </li>
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">VIDEO</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="main">
+      <section class="main__catalog">
+        <ul class="main__list">
+          <li class="main__list-item" data-qa="card">
+            <div class="card">
+              <div class="card__inner">
+                <div class="card__img">
+                  <img
+                    src="./images/imac.jpeg"
+                    alt="imac"
+                    class="card__img card__img--imac"
+                  >
+                </div>
```

Comment:
You gave the same class name 'card__img' for two different tags

Reply:
fix @imaksweb comment, maybe you event don't need additional wrapper behind the image?